esbuild.js

```diff
@@ -1,19 +1,20 @@
-import process from 'node:process';
-import * as esbuild from 'esbuild'
+import process from "node:process";
+import * as esbuild from "esbuild";
 
 const name = process.argv[2];
+const entrypoint = process.argv[3];
 
 function config(opt) {
   return {
-    entryPoints: ['src/index.js'],
-    target: ['esnext'],
-    format: 'esm',
+    entryPoints: [entrypoint],
```

Comment:
```suggestion
    entryPoints: [entrypoint ?? "src/index.js"],
```

then we don't need to pass it in the build calls below

Reply:
ah great idea